### ISSUE TITLE
Add flag to accept raw image data in stdin

### ISSUE THREAD

**Issue by jdiez17** [eXeC64/imv#33]
Example use case: `gpg --decrypt image.png.pgp | imv -r`

**Comment by eXeC64** [comment]
This'll need minor imv_image refactoring.

Useful code example from FreeImage docs:
```FreeImageIO io;
io.read_proc = ReadProc; // pointer to function that calls fread
io.write_proc = NULL;    // not needed for loading
io.seek_proc = SeekProc; // pointer to function that calls fseek
io.tell_proc = TellProc; // pointer to function that calls ftell
FILE *f = fopen(“mybitmap.bmp”, “rb”);
FIBITMAP *bitmap = FreeImage_LoadFromHandle(FIF_BMP, &io, (fi_handle)f, 0);
fclose(f);
if (bitmap) {
    // bitmap successfully loaded!
    FreeImage_Unload(bitmap);
}```

**Comment by eXeC64** [comment]
Solved by #76
